Navidux/Sources/Navidux/Coordinator.swift

```diff
@@ -3,8 +3,6 @@ import UIKit
 /// Main Object that used in navigation. Have many different types of presentation and inner methods to manipulate navigation stack.
 /// Work on states with associated values.
 /// - Note use **actionReducer(action:)** function with some action to change state of navigation stack. It's trigger inner function to change navigation screen.
-/// - Note use **start()** function to trigger first default screen or **actionReducer(action:)** with **restruct** action to set up inital navigation stack state.
 public protocol Coordinator: AnyObject {
```

Comment:
What difference we pump into Navigator name or Coordinator. Maybe if we do `route` use Router name?

Reply:
Good idea about Router, because Coordinator or Navigator is specific navigation layer in architecture. 
We provide abstraction for navigation between screens in other architectures, do is not? If it's ours use case, I think `Router` great idea